blog-service/2024-02-15-ui.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Announcing New and Improved Sumo Logic UI - Closed Beta
+image: https://www.sumologic.com/img/logo.svg
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Update - We've released a Closed Beta of the brand new and improved Sumo Logic user interface (UI). Here are some ways we’ve streamlined and simplified the overall experience:
+
+* Access to all Observability features (like Alerts and and Security features (like Cloud SIEM and Cloud SOAR) in one unified left-navigation menu.
+* Access to administrative features (Administration, Configuration, Help, and User Preferences) have been moved to the top toolbar menu.
+* In-app tabs have been replaced with native browser tabs, meaning faster performance, better usability, and consistent browsing experience similar to other web applications. You can now utilize browser tabs, session history, the back button, favorites, and other native browser capabilities you're familiar with.
+* URLs are now stateful, making it easier for you to share the most up-to-date content with your teammates. Any changes you make in your UI will be reflected globally in the URL parameters.
+
+For more information, contact your Sumo Logic sales representative. In the coming months, we'll release our opt-in open beta to all customers. Stay tuned!
+
+<!-- [Learn more here](/docs/get-started/sumo-logic-ui-new). We'd love your feedback! Just click the **Feedback** button at the bottom of the left-side nav or reach out to our [Sumo Dojo Slack community](https://sumodojo.slack.com/) #sumo-new-ui channel.-->
```

Comment:
@agaur , this is the Kanso release note for Feb 15. As it's a closed beta, can you confirm whether to link to the Kanso-specific doc?  I currently have that doc (below) set to private, meaning only users with the link can access it, and it's not searchable on our site or search indexes.